nncf/experimental/post_training/algorithms/fast_bias_correction/onnx_algo_backend.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import List
+import numpy as np
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.tensor_statistics.collectors import ReductionShape
+from nncf.common.utils.backend import BackendType
+from nncf.common.utils.registry import Registry
+
+from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import LAYERS_WITH_BIAS_METATYPES
+from nncf.experimental.onnx.graph.metatypes.onnx_metatypes import ONNX_OPERATION_METATYPES
+from nncf.experimental.onnx.graph.transformations.commands import ONNXBiasCorrectionCommand
+from nncf.experimental.onnx.graph.transformations.commands import ONNXModelExtractionCommand
+from nncf.experimental.onnx.graph.transformations.commands import ONNXTargetPoint
+from nncf.experimental.onnx.statistics.collectors import ONNXMeanStatisticCollector
+from nncf.experimental.onnx.tensor import ONNXNNCFTensor
+from nncf.experimental.post_training.algorithms.fast_bias_correction.algo_backend import ALGO_BACKENDS
+from nncf.experimental.post_training.algorithms.fast_bias_correction.algo_backend import FBCAlgoBackend
+
+
+@ALGO_BACKENDS.register()
```

Comment:
The idea was to create a enum with the list of backends: `BACKENDS.ONNX, BACKENDS.OV,BACKENDS.PT, BACKENDS.TF` and use their names at registration, like:

@ALGO_BACKENDS.register(BACKENDS.ONNX)

Reply:
Oh, you probably just need to put `BackendType.ONNX` there.